README.md

```diff
@@ -1,2 +1,97 @@
-# budget app
- Web application where you can manage your budget
+# re:Budget
+
+> This project is created as part of the rails capstone at Microverse. re:budget is a web application where a user manage its budget: have a list of transactions associated with a category, so that the user can see how much money it spent and on what.
+
+> Credit of the design goes to [Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/). The Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).
+
+## Demo Link
+
+- [Live App]()
```

Comment:
- [ ] <b>REQUIRED:</b> This project requires to deploy online share the link in the readme, Kindly do so to comply with project requirements.